### ISSUE TITLE
Lists of Bookmarks and Search Results should be an Ordered List

### ISSUE THREAD

**Issue by mtribone** [psu-libraries/cho#424]
The sections for bookmarks and search results visually output what appears to be an ordered list of results, however, the code underneath is a series of `divs` that can be an accessibility issue.  Properly coded lists give users of assistive technology more usable information about the list, where they are in the list, the number of items in the list, and the ability to jump around the the list. It may "look" like a list, but it doesn't function like a list.

Solution is to output the contents for both bookmarks and search results as an ordered list. This may be a bit tricky because it appears that there are several partials that make up the appearance of the content of the list.

**Comment by mtribone** [comment]
![screen shot 2018-03-05 at 2 47 31 pm](https://user-images.githubusercontent.com/4163828/36996268-5980c61c-2084-11e8-9838-007ef14f21b0.png)

**Comment by mtribone** [comment]
![screen shot 2018-03-05 at 2 52 08 pm](https://user-images.githubusercontent.com/4163828/36996422-d3388094-2084-11e8-8d8b-f970c8151c06.png)

**Comment by awead** [comment]
Blocked by #467

**Comment by mtribone** [comment]
I've sent an email with questions to the Blacklight-dev Google group about the search results and bookmarks and have sent an email to the Accessibility group at Penn State for a consultation.

**Comment by mtribone** [comment]
We've tested the demo app of Blacklight 7 with Bootstrap 4 with Michele and the accessibility group and the appearance of the ordered list did not trip her up. Although we did find other issues.

Decision is to move forward and only change if other stakeholders find issues with the current implementation.

**Comment by ntallman** [comment]
Closing this, it can be reopened in future if stakeholders have issues.
